vault/identity_store_entities.go

```diff
@@ -446,6 +446,26 @@ func (i *IdentityStore) handleEntityReadCommon(entity *identity.Entity) (*logica
 	// formats
 	respData["aliases"] = aliasesToReturn
 
+	// Fetch the groups this entity belongs to and return their identifiers
+	groups, impliedGroups, err := i.groupsByEntityID(entity.ID)
```

Comment:
minor: Can we match the external names with the internal variable names? implied vs inherited